projects/packages/search/src/initializers/class-initializer.php

```diff
@@ -7,14 +7,187 @@
 
```

Comment:
Would it be worthwhile to update our WPCOM initializer to follow this pattern? Also, would there be any backward compatibility issues with the WPCOM initializer?

Reply:
> Would it be worthwhile to update our WPCOM initializer to follow this pattern?

I guess we could, but not an priority at the moment.

>  Also, would there be any backward compatibility issues with the WPCOM initializer?

I kept the `intialize` function to keep the compatibility. I'll test on WPCOM and see if it breaks anything.

Reply:
Tested on WPCOM, luckily it works 😄